datahub-web-react/src/CustomThemeProvider.tsx

```diff
@@ -12,8 +13,12 @@ interface Props {
 
 const CustomThemeProvider = ({ children, skipSetTheme }: Props) => {
     const [currentTheme, setTheme] = useState<Theme>(defaultThemeConfig);
+    const customThemeId = loadThemeIdFromLocalStorage();
 
     useEffect(() => {
+        // use provided customThemeId and set in useSetAppTheme.tsx if it exists
```

Comment:
a bit confused, where does the custom theme id actually end up being used?

Reply:
left you a comment in slack!